Sources/StreamChatUI/ChatMessageList/ChatMessage/ChatMessageContentView.swift

```diff
@@ -1014,12 +1014,12 @@ extension ChatMessageContentView {
     }
     
     @available(*, deprecated, renamed: "createOnlyVisibleToYouImageView")
-    open func createOnlyVisibleForYouIconImageView() -> UIImageView {
+    public func createOnlyVisibleForYouIconImageView() -> UIImageView {
```

Comment:
Why this change? It can be a breaking change

Reply:
Xcode 14 raised a warning here. If this is not the right solution, please let me know what's the best way to fix it.

Reply:
We need to keep it as open, otherwise our integrators cannot override those methods. We can tackle these warnings later on

Reply:
The warning is saying that there is no open functions in extensions. So right now this is not working. Or we move this deprecation in the actual file, or we just leave it public. If no one compplained about this yet, it means nobody is using it anyways

Reply:
Oh ok! Let's move to public then